runtime/etc/src/mli/common_code.c

```diff
@@ -133,15 +117,14 @@ const char* chpl_mli_errstr(enum chpl_mli_errors e) {
   static const char* mli_errors_[] = {
     "NONE",
     "SHUTDOWN",
-    "EUNKNOWN",
-    "ENOFUNC",
-    "ESOCKET",
-    "EEXCEPT",
-    "EMEMORY"
+    "UNKNOWN",
+    "NOFUNC",
+    "SOCKET",
+    "MEMORY"
   };
 
-  if (e > CHPL_MLI_CODE_NONE || e < CHPL_MLI_CODE_EMEMORY) {
-    return "INVALID_ERROR_CODE";
+  if (e > CHPL_MLI_CODE_NONE || e < CHPL_MLI_CODE_MEMORY) {
+    return "<invalid>";
```

Comment:
Should there be an error for using the number that used to correspond to EEXCEPT?

Reply:
Just closed the gap in the enum :)